### PR TITLE
PSASSET-832-RI Reservation Coverage

### PR DESCRIPTION
**Pull request by rshade**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/reserved_instances/coverage/reserved_instance_coverage.pt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+##################
+
+parameter "param_email" do
+  category "Contact"
+  label "Email addresses (separate with commas)"
+  type "string"
+  allowed_pattern /^([a-zA-Z0-9-_.]+[@]+[a-zA-Z0-9-_.]+[.]+[a-zA-Z0-9-_]+,*|)+$/
+end
+
+#parameter "param_service" do
```
Comment (gonzalez) [inline cost/aws/reserved_instances/coverage/reserved_instance_coverage.pt]: needs cleaning up

Comment (tusharshahrs) [reply cost/aws/reserved_instances/coverage/reserved_instance_coverage.pt]: cleaned up

**Comment by rshade** [comment]
Add it to the root reademe

**Comment by tusharshahrs** [comment]
Added it to root readme

**Comment by tusharshahrs** [comment]
@rshade added it to root readme

**Comment by rshade** [comment]
@tusharshahrs  your code is fine. You can approve it and merge it

**Pull request merged**